Build failure on intel_s1000_crb board.

**Describe**
Couldn't build test cases on i=intel_s1000_crb board, it would report a build error.

**To Reproduce**
Steps to reproduce the behavior:
1.  west build -b intel_s1000_crb -p auto samples/hello_world/
2.  See error



**Screenshots or console output**
In file included from /home/chenpeng/zephyrproject/zephyr/kernel/queue.c:20:
..**/kernel/include/ksched.h:17:43: error: too few arguments provided to function-like macro invocation
             >= K_HIGHEST_APPLICATION_THREAD_PRIO);**
                                                 ^
**../include/toolchain/common.h:144:9: note: macro 'BUILD_ASSERT' defined here
#define BUILD_ASSERT(EXPR, MSG)**                          \
        ^
In file included from /home/chenpeng/zephyrproject/zephyr/kernel/queue.c:20:
../kernel/include/ksched.h:16:1: error: type specifier missing, defaults to 'int' [-Werror,-Wimplicit-int]
BUILD_ASSERT(K_LOWEST_APPLICATION_THREAD_PRIO
^
2 errors generated.


**Environment (please complete the following information):**
 - OS: fedora28
 - Toolchain: xcc
 - Commit ID: 1134edd6ad1d

--- a/include/toolchain/common.h
+++ b/include/toolchain/common.h
@@ -141,7 +141,7 @@
 /* Compile-time assertion that makes the build to fail.
  * Common implementation swallows the message.
  */
-#define BUILD_ASSERT(EXPR, MSG)				 \
+#define BUILD_ASSERT(EXPR, MSG...) \
 	enum _CONCAT(__build_assert_enum, __COUNTER__) { \
 		_CONCAT(__build_assert, __COUNTER__) = 1 / !!(EXPR) \
 	}

